frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -198,6 +219,17 @@ pub mod pallet {
     pub(crate) type PreApprovedDevelopers<T: Config> =
         StorageMap<_, Twox64Concat, T::AccountId, (), ValueQuery>;
 
+    /// Nomination transfer charges cooldowns ending blocks.
```

Comment:
Please elaborate why this delay really required, is balance migrations have some I calculated weights?

Reply:
Discussed in slack - once we agree how to proceed, I will update (probably remove) this part.